orcid-core/src/main/java/org/orcid/core/manager/read_only/impl/AffiliationsManagerReadOnlyImpl.java

```diff
@@ -124,8 +123,7 @@ public EmploymentSummary getEmploymentSummary(String userOrcid, Long employmentI
      * @return the list of employments that belongs to this user
      * */
     @Override
-    @Cacheable(value = "employments-summaries", key = "#userOrcid.concat('-').concat(#lastModified)")
```

Comment:
This one - affiliationManagerReadOnly.getEmploymentSummaryList - calling different method from v3 because I forgot to update the dao method it calls, where cache was moved to. Now brought in line with v3